Tensile/Common.py

```diff
@@ -156,6 +156,7 @@
 globalParameters["MaxDepthU"] = 256               # max DepthU value to allow
 globalParameters["ShortNames"] = False            # on windows kernel names can get too long; =True will convert solution/kernel names to serial ids
 globalParameters["MergeFiles"] = True             # F=store every solution and kernel in separate file; T=store all solutions in single file
+globalParameters["MaxFileName"] = 128 # If a file name would be longer than this, shorten it with a hash.
```

Comment:
How about setting this to `64`? It seems like `128` will still make the name too long.

Reply:
rocBLAS build with this PR has a maximum file name length of 143.  In a docker container the limit seems to be around 224.  If any utility adds filename overhead as large as 80, that utility needs work.  Back to your issue.  I'm sure you can set your own MaxFileName to 64.  The default value of 128 appears to be quite sensible.

Reply:
rocBLAS build with this PR has a maximum file name length of 143.  In a docker container the limit seems to be around 224.  If any utility adds filename overhead as large as 80, that utility needs work.  Back to your issue.  I'm sure you can set your own MaxFileName to 64.  The default value of 128 appears to be quite sensible.

Reply:
A rocBLAS build with this PR has a maximum file name length of 143.  In a docker container the limit seems to be around 224.  If any utility adds filename overhead as large as 80, that utility needs work.  Back to your issue.  I'm sure you can set your own MaxFileName to 64.  The default value of 128 appears to be quite sensible.

Reply:
A rocBLAS build with this PR has a maximum file name length of 143.  In a docker container the limit seems to be around 224.  If any utility adds filename overhead as large as 80, that utility needs work.  Back to your issue.  I'm sure you can set your own MaxFileName to 64.  The default value of 128 appears to be quite sensible.

Reply:
I am seeing a max file length of 277, which is too large. How do I set my own `MaxFileName` without forking Tensile?